subsys/logging/log_core.c

```diff
@@ -352,6 +359,43 @@ void log_n(const char *str,
 	}
 }
 
+int log_format_set(char *backend_name, uint32_t log_type)
+{
+	const struct log_backend *backend = log_backend_get_by_name(backend_name);
+
+	if (log_type >= ARRAY_SIZE(log_format_table)) {
+		return -EINVAL;
+	}
+
+	if (backend != NULL) {
+		if (backend->api->format_set == NULL) {
+			return -ENOTSUP;
+		} else {
+			return backend->api->format_set(backend, log_type);
```

Comment:
The field "format_set" does not yet exist. It is currently added in the next commit.

Reply:
Added the declaration of format_set in the same commit now.